portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm

```diff
@@ -17,27 +17,37 @@
     #end
 
     <li class="site-list-item py-1 $!{currentSiteClass}">
-        <div class="site-list-item-head d-flex align-items-stretch pe-2">
-            <button class="btn btn-nav btn-site rounded-end text-start border-0 me-1 d-flex flex-grow-1 #if (!$isCurrentSite)collapsed#end"
-                    data-bs-toggle="collapse"
-                    data-bs-target="#site-${index}-pages"
-                    aria-expanded="${isExpanded}"
-                    aria-controls="site-${index}-pages">
-                <i class="bi bi-chevron-right" aria-hidden="true"></i>
-                <span>${site.title}</span>
-            </button>
+        <div class="site-list-item-head d-flex align-items-center pe-2">
+            <div>
```

Comment:
Is this 'div' required?

Reply:
No. None of those divs I added were needed actually. Well spotted, thanks.

Reply:
No problem :)